draft-ietf-6man-slaac-renum-08.xml

```diff
@@ -539,6 +664,9 @@ In particular:
 Routers MAY be explicitly or implicitly configured to send multiple RAs and split information between them. For example, a router could be configured to send information associated with different provisional domains <xref target="RFC7556"/> in different RAs, or to send multiple RAs, one per VRRPv3 <xref target="RFC9568"/> group.
 </t>
 <t>
+If prefix-specific link-local addresses [draft-ietf-6man-slaac-renum] are enabled for the interface, the router SHOULD include one PIO into each RA, as described in [draft-ietf-6man-slaac-renum].
```

Comment:
One aspect that seems to be missing is how the RA "sending rules" would apply. -- If you want to keep the same RA packet rate, then the RA frequency for each PIO would be reduced.  OTOH, if you want the same RA frequency for each PIO, then the packet rate would increse.  

IOW, how would this mechanisms affect the RA rate?

Reply:
The text below talks about sending them all together, so the same sending rules apply. I've just updated the text slightly, PTAL